test/itowns-testing.js

```diff
@@ -24,18 +24,14 @@ global.fetch = function _fetch(url) {
 
     // try reading as a file
     fs.readFile(process.env.PWD + url, 'utf-8', (err, content) => {
-        if (!content) {
+        if (!content || err) {
```

Comment:
Because this mock implements fetch by: 
- first testing if it's a local file (Ortho.json for instance) in which case its content is returned
- then emulating a network fetch